README.md

```diff
@@ -37,13 +39,37 @@ If there is anything wrong with this repository (documentation needed / broken t
       1. real 2m32.230s
       2. user 0m0.000s
       3. sys 0m0.016s
-5. Bring up the VirtualBox UI and Show the `eagle-development-environment`
+   3. For example, one machine was ready after the terminal showed the following message:
+      
+      *VirtualBox Guest Additions: Building the modules for kernel 6.5.0-21-generic.
+      update-initramfs: Generating /boot/initrd.img-6.5.0-21-generic
+      VirtualBox Guest Additions: Running kernel modules will not be replaced until 
+      the system is restarted or 'rcvboxadd reload' triggered
+      VirtualBox Guest Additions: reloading kernel modules and services
+      VirtualBox Guest Additions: kernel modules and services 7.0.14 r161095 reloaded
+      VirtualBox Guest Additions: NOTE: you may still consider to re-login if some
+      user session specific services (Shared Clipboard, Drag and Drop, Seamless or
+      Guest Screen Resize) were not restarted automatically*
+
+5. Bring up the VirtualBox UI and click the `eagle-development-environment` VM.
 6. Login to the Virtual Machine
    1. uname: `vagrant`
    2. password: `vagrant`
 
+
 **Note** You can alter the vb.memory value in the Vagrantfile based on how much RAM your personal machine has. I wouldn't give it more than %50 of what your system max is
 
+## VM Usage Guide
+1. To start the machine run the following command from within the directory of this cloned repository (i.e. `/eagle-development-environment/`):
+   `vagrant up`
+2. To open Mission  Planner:
```

Comment:
Might want to add separate sections detailing guides for each of the tools we're using (Mission Planner, MAVProxy, ArduPilot, etc.)

Reply:
I will add as I keep compiling info